easysnmp/interface.c

```diff
@@ -1352,12 +1352,31 @@ static int py_netsnmp_attr_string(PyObject *obj, char *attr_name, char **val,
             int retval;
 
 #if PY_MAJOR_VERSION >= 3
-            // Encode the provided attribute using latin-1 into bytes and
-            // retrieve its value and length
-            PyObject *attr_bytes = PyUnicode_AsEncodedString(attr, "latin-1",
-                                                             "surrogateescape");
+
+            PyObject *attr_bytes;
+            if(PyBytes_CheckExact(attr))
```

Comment:
style bug: `if(PyBytes_CheckExact(attr))` should be `if (PyBytes_CheckExact(attr))`